### PR TITLE
add game-pages

### PR DESCRIPTION
**Pull request by noeybynk**
create new branch name "game-davinci-apps" for creating game's pages

### CODE REVIEW

File: game/template/easy_level1.html [hunk 1]
```diff
@@ -1,63 +1,25 @@
...
-}
-/* .button2 {background-color: #008CBA;} Blue */
-/* .button3 {background-color: #f44336;} Red  */
-/* .button4 {background-color: #e7e7e7; color: black;} Gray  */
-/* .button5 {background-color: #555555;} Black */
-h1 {
-  co
-}
-</style>
+  <title>DAVINCI GAME</title>
```
Comment (SukritaEarn) [inline game/template/easy_level1.html]: I think the title should be our game name "Riddle Me This" or Easy stage.

File: game/template/easy_level1.html [hunk 2]
```diff
@@ -1,63 +1,25 @@
...
 
-<!-- <h2>Button Colors</h2>
-<p>Change the background color of a button with the background-color property:</p> -->
-
 <!-- <button class="button">Green</button> -->
-<a href= "https://www.facebook.com/" class="button-left">Linktofacebook</a>
-<a href= "https://www.youtube.com/" class="button-right">Linktoyoutube</a>
+<a href= "https://www.facebook.com/" class="button-left">Main Menu</a>
+<a href= "https://www.youtube.com/" class="button-right">Next</a>
 <h1>DAVINCI GAME</h1>
```
Comment (SukritaEarn) [inline game/template/easy_level1.html]: I think the header should be change into the game question number and specify the stage such as Easy 1.

File: game/template/easy_level1.html (hunk 1 above)
Comment (SukritaEarn) [inline game/template/easy_level1.html]: The title should be our game name "Riddle Me This" or "Easy stage".

File: game/template/medium-level/medium_level4.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    <head>
+        <title>DAVINCI GAME</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <link rel="stylesheet" href="medium.css">
+    </head>
+
+    <body>
+        <a href= "_____________" class="button-left">Main menu</a>
+        <h1>DAVINCI GAME</h1>
```
Comment (SukritaEarn) [inline game/template/medium-level/medium_level4.html:12]: This should be question number with a stage such as Medium 1.

File: game/template/medium-level/medium_level4.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>DAVINCI GAME</title>
```
Comment (SukritaEarn) [inline game/template/medium-level/medium_level4.html:4]: This should be "Medium stage".

File: game/template/medium-level/medium_level1.html [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+        <hr>
+        
+        <img src="4-billion.png" class="img" alt="billion" height="250" width="300">
+        
+        <div class="input-box">
+            <input type="fullword" placeholder="text here">
+            <button class="button-hint">Hint</button>
+            <form>
+                <INPUT TYPE="button" class="button-submit" VALUE="SUBMIT" onClick="parent.location='medium-level1.html'">
```
Comment (KasidisGit) [inline game/template/medium-level/medium_level1.html:22]: This section should render with django context.

**Pull request merged**